libcaf_net/caf/net/actor_shell.test.cpp

```diff
@@ -144,7 +144,7 @@ TEST("actor shells can receive messages") {
   auto app = app_t::make(actor{self});
   auto transport = net::octet_stream::transport::make(fd1, std::move(app));
   auto mgr = net::socket_manager::make(&mpx, std::move(transport));
-  mpx.start(mgr);
+  std::ignore = mpx.start(mgr);
```

Comment:
Why discard the result instead of `check(mpx.start(mgr))` or even `require(mpx.start(mgr))`?